Makefile

```diff
@@ -147,17 +177,38 @@ rpm: $(CR_BIN) $(PKG_INFO)
 .PHONY: docker-rpm
 docker-rpm:
 	$(call print_msg, Running ClusterRunner Docker RPM builder... )
-	$(eval TAG := productivity/clusterrunner)
+	docker build -t $(DOCKER_TAG) -f Dockerfile .
```

Comment:
Is the -f necessary if this is the only Dockerfile in the repo?

Reply:
No its not, but I like being extra explicit in Makefiles since reader might not be familiar with Docker